src/test/run-make/issue-71519/Makefile

```diff
@@ -2,6 +2,7 @@ include ../../run-make-fulldeps/tools.mk
 
 # ignore-msvc
 # needs-rust-lld
```

Comment:
Why is this not covered by needs-rust-lld? If lld does not support s390x, we probably shouldn't be shipping a rust-lld either?

Reply:
Oh, I guess having rust-lld can still make sense for cross-compiling.

Reply:
Exactly.  Specifically for linking wasm binaries, where lld is really the only option.